cache/search_stats.go

```diff
@@ -264,8 +276,18 @@ func (ssc *SearchStatsCacheImpl) isClassWithUnits(class, uid string, count int,
 
 	if h, ok := stats[uid]; ok {
 		for i := range cts {
-			if c, ok := h[cts[i]]; ok && c >= count {
-				return true
+			if c, ok := h[cts[i]]; ok {
+				minOk := minCount == nil
+				maxOk := maxCount == nil
+				if minCount != nil {
```

Comment:
Consider using int with -1 as non valid default value

Reply:
Return 
minCount == -1 || minCount <= c 
and
Similar for max count... One liner.

Reply:
i think using pointer is more idiomatic

Reply:
Still just write a one liner

(max == nil || c <= max) and (min == nil || c >= min)

Reply:
done